CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 ## 4.52.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_secret_manager_secret_version_access` ([#5147](https://github.com/hashicorp/terraform-provider-google-beta/pull/5147))
+* **New Resource:** `google_cloudbuildv2_connection` (beta) ([#5140](https://github.com/hashicorp/terraform-provider-google-beta/pull/5140))
```

Comment:
```suggestion
* **New Resource:** `google_cloudbuildv2_connection` ([#5140](https://github.com/hashicorp/terraform-provider-google-beta/pull/5140))
```

I think we remove the `(beta)` from the final changelog (there are a couple others that I'm not marking separately.)